main/EDMainMod.F90

```diff
@@ -15,6 +15,7 @@ module EDMainMod
   use FatesInterfaceMod        , only : hlm_current_day 
   use FatesInterfaceMod        , only : hlm_use_planthydro 
   use FatesInterfaceMod        , only : hlm_reference_date
+  use FatesInterfaceMod        , only : hlm_use_ed_prescribed_phys
   use FatesInterfaceMod        , only : hlm_use_ed_st3 
```

Comment:
Not part of this pull request I know, but 'use_ed_st3' is quite an arcane varaible name... Think we're going to lose some people with that one.

Reply:
Im fine with that, can you create an issue, or propose some changes here?  Are there others you would like to change as well?